src/Arcus.Security.Tests.Unit/Core/Configuration/IHostBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Threading.Tasks;
+using Arcus.Security.Core;
+using Arcus.Security.Core.Storage.Source;
+using Arcus.Security.Tests.Core.Fixture;
+using Arcus.Security.Tests.Unit.Core.Stubs;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace Arcus.Security.Tests.Unit.Core.Configuration
+{
+    public class IHostBuilderExtensionsTests
+    {
+        [Fact]
+        public async Task ConfigureSecretStore_AddInMemorySecretProvider_UsesInMemorySecretsInSecretStore()
+        {
+            // Arrange
+            string secretKey = "MySecret";
+            string secretValue = $"secret-{Guid.NewGuid()}";
+            var stubProvider = new InMemorySecretProvider((secretKey, secretValue));
```

Comment:
This one is just for testing and not part of our NuGet packages, right?

Reply:
Yep, right, it's under `Arcus.Security.Unit/Stubs/InMemorySecretProvider.cs`.